### ISSUE TITLE
Genie.Renderer.Html Not Finding Consts Defined in Genie.Renderer

### ISSUE THREAD

**Issue by drippdropp** [GenieFramework/Genie.jl#193]
**Describe the bug**
When I call an endpoint with `html` function in controller, I get an error: 
`UndefVarError: VIEWS_FOLDER not defined`

I followed the basic guide and created a new controller using `newcontroller` function, called `DistributionController` and a view called `binomial.jl.html`. The view file just has a single `<h1>` tag in it right now.

I did use `fullstack=true` when creating the app (if that offers any insight).  Also, if I don't use `html` function and just output tags, it works fine and renders.

Here is the file tree: (app is called "Foundations", that's root of the Genie project)
````
λ ~/apps/Foundations/app/resources/distribution/ tree .
.
├── DistributionController.jl
└── views
    └── binomial.jl.html

1 directory, 2 files
````

Here is DistributionController:
````
module DistributionController
  using Genie.Renderer.Html

  function binomial()
    html(:distribution, :binomial, layout = :app)
  end
end
````
I also tried including `using Genie.Renderer` in addition to `using Genie.Renderer.Html` to see if that would help -- no dice.

It's worth noting that if I don't including the `layout` parameter in `html(...)`, then I get: `DEFAULT_LAYOUT_FILE not defined` (also defined in `Renderer`) since that is checked before `VIEWS_FOLDER`.

It appears that in when `html` from `Genie.Renderer.Html` is executed, https://github.com/GenieFramework/Genie.jl/blob/8b53239e1767e0b30e1e663b2a0f8f9b3f1957fb/src/renderers/Html.jl#L322, it is not finding the constant `VIEWS_FOLDER` defined in `Genie.Renderer`. I can see that on line 6 of Html.jl, it is importing `Genie.Renderer`, so I would expect that the value would be found.

I'm still pretty new to Julia, so I haven't tried re-defining `VIEWS_FOLDER` in `Html.jl` and re-compiling to see if it resolves the issue (also since from looking at the code, it seems like it should be working) and I haven't found any other references to this issue from a cursory scan of the interwebs, so seems like this isn't widespread. Maybe this is a Julia 1.3 thing? Like a difference in how `import` works? Not sure if it is something I have done wrong or if there is some sort of path issue, or otherwise.

**Error stacktrace**
````
######################### ERROR STACKTRACE #########################
UndefVarError: VIEWS_FOLDER not defined
#html#13(::Symbol, ::Module, ::Int64, ::Dict{String,String}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Genie.Renderer.Html.html), ::Symbol, ::Symbol) at /Users/USERNAME_GOES_HERE/.julia/packages/Genie/Pe2DP/src/renderers/Html.jl:322
#html at ./none:0 [inlined]
binomial() at /Users/USERNAME_GOES_HERE/apps/Foundations/app/resources/distribution/DistributionController.jl:8
#invokelatest#1 at ./essentials.jl:709 [inlined]
invokelatest at ./essentials.jl:708 [inlined]
match_routes(::HTTP.Messages.Request, ::HTTP.Messages.Response, ::Nothing, ::Genie.Router.Params{Any}) at /Users/USERNAME_GOES_HERE/.julia/packages/Genie/Pe2DP/src/Router.jl:502
route_request(::HTTP.Messages.Request, ::HTTP.Messages.Response, ::Sockets.IPv4) at /Users/USERNAME_GOES_HERE/.julia/packages/Genie/Pe2DP/src/Router.jl:130
handle_request(::HTTP.Messages.Request, ::HTTP.Messages.Response, ::Sockets.IPv4) at /Users/USERNAME_GOES_HERE/.julia/packages/Genie/Pe2DP/src/AppServer.jl:92
run_work_thunk(::Genie.AppServer.var"#14#15"{HTTP.Messages.Request,HTTP.Messages.Response}, ::Bool) at /Users/USERNAME_GOES_HERE/.julia/packages/Genie/Pe2DP/src/AppServer.jl:83
#remotecall_fetch#144(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Distributed.remotecall_fetch), ::Function, ::Distributed.LocalProcess) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.3/Distributed/src/remotecall.jl:375
remotecall_fetch(::Function, ::Distributed.LocalProcess) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.3/Distributed/src/remotecall.jl:375
#remotecall_fetch#148(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Distributed.remotecall_fetch), ::Function, ::Int64) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.3/Distributed/src/remotecall.jl:417
remotecall_fetch(::Function, ::Int64) at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.3/Distributed/src/remotecall.jl:417
setup_http_handler(::HTTP.Messages.Request, ::HTTP.Messages.Response) at /Users/USERNAME_GOES_HERE/.julia/packages/Genie/Pe2DP/src/AppServer.jl:107
setup_http_handler at /Users/USERNAME_GOES_HERE/.julia/packages/Genie/Pe2DP/src/AppServer.jl:106 [inlined]
#6 at /Users/USERNAME_GOES_HERE/.julia/packages/Genie/Pe2DP/src/AppServer.jl:59 [inlined]
handle at /Users/USERNAME_GOES_HERE/.julia/packages/HTTP/lZVI1/src/Handlers.jl:253 [inlined]
handle(::HTTP.Handlers.RequestHandlerFunction{Genie.AppServer.var"#6#12"}, ::HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}) at /Users/USERNAME_GOES_HERE/.julia/packages/HTTP/lZVI1/src/Handlers.jl:276
#4 at /Users/USERNAME_GOES_HERE/.julia/packages/HTTP/lZVI1/src/Handlers.jl:345 [inlined]
macro expansion at /Users/USERNAME_GOES_HERE/.julia/packages/HTTP/lZVI1/src/Servers.jl:360 [inlined]
(::HTTP.Servers.var"#13#14"{HTTP.Handlers.var"#4#5"{HTTP.Handlers.RequestHandlerFunction{Genie.AppServer.var"#6#12"}},HTTP.ConnectionPool.Transaction{Sockets.TCPSocket},HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}})() at ./task.jl:333
````
**To reproduce**
I just hit an endpoint and I get the error stack shown above.

**Expected behavior**
Load the view?

**Additional context**
julia> versioninfo()
Julia Version 1.3.0
Commit 46ce4d7933 (2019-11-26 06:09 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin19.0.0)
  CPU: Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_REVISE = auto

**Comment by essenciary** [comment]
Thanks @drippdropp for reporting this. It's probably an issue in Genie's rendering layer, I've been tweaking a lot there lately. I'll take a look in a couple of hours and get back to you.

**Comment by essenciary** [comment]
@drippdropp Looked into it - that part fell victim to a recent massive refactoring in the rendering layer. Some constants didn't get properly updated. I'm surprised the tests passed - looks like these methods weren't covered. 

Anyway, I submitted a registration request for v0.24.1 - that should address the problem and you code should just work (with and without `layout = :app`). The new version should be up soon.

**Comment by drippdropp** [comment]
Thank you for the quick turn around @essenciary!

**Comment by essenciary** [comment]
@drippdropp No problem - should be fixed now on the latest 0.24.2. Let me know if that is not the case. Closing for now - please feel free to reopen if needed. Cheers!
